paper/paper.Rmd

```diff
@@ -1,35 +1,42 @@
 ---
-title: "datawizard: An R Package for Easy Data Wrangling"
+title: "datawizard: An R Package for Easy Data Preparation and Statistical Transformations"
 tags:
   - R
   - easystats
 authors:
 - affiliation: 1
-  name: Dominique Makowski
-  orcid: 0000-0001-5375-9967
-- affiliation: 2
   name: Indrajeet Patil
   orcid: 0000-0003-1995-6531
+- affiliation: 2
+  name: Dominique Makowski
+  orcid: 0000-0001-5375-9967
 - affiliation: 3
   name: Mattan S. Ben-Shachar
   orcid: 0000-0002-4287-4801
 - affiliation: 4
   name: Brenton M. Wiernik
   orcid: 0000-0001-9560-6336
 - affiliation: 5
+  name: Etienne Bacher
+  orcid: 0000-0002-9271-5075 
+- affiliation: 6
   name: Daniel Lüdecke
   orcid: 0000-0002-8895-3206
+  
 affiliations:
 - index: 1
-  name: Nanyang Technological University, Singapore
+  name: esqLABS GmbH, Germany
 - index: 2
-  name: Center for Humans and Machines, Max Planck Institute for Human Development, Berlin, Germany  
+  name: Nanyang Technological University, Singapore
 - index: 3
   name: Ben-Gurion University of the Negev, Israel
 - index: 4
-  name: Department of Psychology, University of South Florida, USA 
+  name: Facebook
```

Comment:
```suggestion
  name: Independent Researcher
```

Reply:
Add an author note for me: "Brenton Wiernik is currently an independent researcher and Research Scientist at Meta,  Demography and Survey Science. The current research was in an independent capacity."